pepper-apis/dsm-core/src/main/resources/liquibase/DDP-7344_preFilter.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="DDP-7344" author="simone">
+        <addColumn tableName="ddp_instance">
```

Comment:
should be here only one column add? as I saw from the code there is something like `queryItems`?

Reply:
Yes I also have the same question